src/lightning_lite/strategies/fairscale.py

```diff
@@ -180,15 +178,6 @@ def register_strategies(cls, strategy_registry: Dict) -> None:
             description=cls.__class__.__name__,
         )
 
-    def _reinit_optimizers_with_oss(self, optimizers: List[Optimizer]) -> List["OSS"]:
```

Comment:
Can you also delete the definitions in `sharded` and `sharded_spawn` to use the function below?

Reply:
I'm hesitant to do so. It would mean touching src/pytorch_lightning, increasing the codeowner list.

Reply:
You are still blocked by more CODEOWNERs after two approvals, so might as well go for it :)

Reply:
@awaelchli It shouldn't too much. For strategy-related changes, you should be fine with @carmocca and myself as reviewers (as I am codeowner for strategies and we only need one codeowner approval and two in total).

Reply:
NVM my comment above. They are only marked as codeowner because of the comments in the PL tests that will be reverted. Feel free to resolve and I'll do this in a follow-up

Reply:
Codeowners are no longer a blocker